### PR TITLE
Add FOFB acquisition core GUIs

### PR DESCRIPTION
**Pull request by anacso17**
This PR:
- moves AFC acquisition core trigger windows (Physical and Logical Triggers) to common subpackage
- changes Physical Triggers description to be the high level trigger name related to the CRT channel
- adds first version of individual GUI for SYSID and LAMP acquisition cores
- fixes a bug left in FOFB controllers detail window (leds in TimeFrameLen tab)
- cleanup code

SYSID acquisition:
Screenshot from 2023-07-26 09-03-40
Screenshot from 2023-07-26 09-03-43
Screenshot from 2023-07-26 09-03-46

LAMP acquisition:
Screenshot from 2023-07-12 17-38-34

### CODE REVIEW

**Review by murilobalves — APPROVED** [review]

File: pyqt-apps/siriushla/common/afc_acq_core/triggers.py [hunk 1]
```diff
@@ -33,7 +35,9 @@ def setupui(self):
 
     def get_trigger_groupbox(self, idx):
         trig = 'TRIGGER{0:d}'.format(idx)
-        name = trig + (': ' + self.names[idx] if self.names else '')
+        hltrig = 'Monit' if idx == 7 else HLTimeSearch.get_hl_from_ll_triggers(
+            self.afctiming.substitute(propty_name=f'CRT{idx}'))
```
Comment (ericonr) [inline pyqt-apps/siriushla/common/afc_acq_core/triggers.py:39]: Is there a TODO item for a high level trigger for Monit?

**Review by fernandohds564 — APPROVED** [review]

File: pyqt-apps/siriushla/common/afc_acq_core/triggers.py (hunk 1 above)
Comment (anacso17) [reply pyqt-apps/siriushla/common/afc_acq_core/triggers.py:39]: Yes @ericonr ! I talked offline to @fernandohds564 and we plan to implement two new high level triggers: SI-Fam:TI-BPM-MonitClk (AMC7) and SI-Fam:TI-BPM-OrbIntlk (AMC4)

**Review by ericonr — APPROVED** [review]
Just one suggestion: it would be nice to add buttons to open the trigger windows directly to the BPM acquisition window as well.

**Comment by anacso17** [comment]
> Just one suggestion: it would be nice to add buttons to open the trigger windows directly to the BPM acquisition window as well.

Done in https://github.com/lnls-sirius/hla/pull/645!

**Pull request merged**